From c7be08b106dc945dc3baa567457bf4e8d1d0dc8c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 15 Oct 2020 17:14:53 +0300
Subject: [PATCH] [SW-12754] gaudi: security: remove finished todos

TODOs related to the RRs initialization have been handled already, so
remove them.
No functional changes.

Change-Id: I523fb656138a2ea95e7059e2b584061042f0a8d8
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index ccf2149..ba46cc8 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -490,9 +490,6 @@
 
 	prot_base += SNGL_REG_PROT_SIZE;	/* offsets: 0x280 - 0xF80 */
 	set_prot_bits_range_all(prot_base, PROT_SIZE_MAX - (prot_base - base));
-
-	/* Protect CPU_TIMESTAMP - 4K */
-	/* TODO: use RRs for CPU_TIMESTAMP protection */
 }
 
 static void preboot_secure_pll(void)
@@ -731,11 +728,6 @@
 		BIT(PSOC_GLOBAL_CONF_PROT_AW_SHIFT);
 	regmap_writel(mask, mmPSOC_GLOBAL_CONF_PROT);
 
-	/*
-	 * TODO: use RRs to protect:
-	 *	 I2C M0/1, I2C_S, SPI, UART0-1, TIMER, WDOG
-	 */
-
 	/* Note: PSOC_TIMESTAMP should not be protected */
 
 	/* Protect EFUSE */
@@ -751,15 +743,6 @@
 
 	/* Protect GLOBAL_CONF - 4K */
 	preboot_secure_psoc_global_conf();
-
-	/*
-	 * TODO: use RRs to protect:
-	 *	 GPIO0-6, BTL, CS_TRACE, DFT_EFUSE, RPMs,
-	 *	 PSOC PLLSs(CPU, MME, PCI, TPC, HBM), PM, TS, PWM0-3
-	 *	 RAZWI
-	 * and
-	 *	 PCIE_PMA_0-3
-	 */
 
 	/* TODO: handle PSOC_DBG/PSOC_ETR/AXICTL */
 }
@@ -800,7 +783,7 @@
 	/* Protect PCIE_WRAP - 4K */
 	preboot_secure_pci_wrap();
 
-	/* TODO: use RRs to protect: DBI, CORE, AUX, PHY */
+	/* The rest is protected by RRs */
 }
 
 #define MMES_NUM	4
